Dalamud.DiscordBridge/Model/QueuedRetainerItemSaleEvent.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Dalamud.DiscordBridge.XivApi;
+using Dalamud.Game.Chat;
+using Dalamud.Game.Chat.SeStringHandling;
+
+namespace Dalamud.DiscordBridge.Model
+{
+    public class QueuedRetainerItemSaleEvent : QueuedXivEvent
+    {
+        public SeString Message { get; set; }
```

Comment:
add the Item row here, populate it in DiscordMessageQueue and pass it to DiscordHandlers

Reply:
DiscordMessageQueue is already handling it https://github.com/reiichi001/Dalamud.DiscordBridge/blob/master/Dalamud.DiscordBridge/DiscordMessageQueue.cs#L88 

In DiscordMessageQueue, I currently process the Item payload because there are multiple things we'd want to get out of it anyways, like number of items (to do) and whether it was HQ (sent, but possibly not needed as the message has this already handled), which is then sent to the handler.